kafka/event.go

```diff
@@ -145,6 +145,33 @@ func (o OAuthBearerTokenRefresh) String() string {
 	return "OAuthBearerTokenRefresh"
 }
 
+// NewApplicationRebalanceError reports an application-level error occuring during a RebalanceCb call.
+func NewApplicationRebalanceError(wrapped error) *ApplicationError {
+	return &ApplicationError{
+		Err: fmt.Errorf("rebalance error: %w", wrapped),
```

Comment:
`"Application rebalance error: %w"` perhaps, to make it obvious where this error originates from.

Reply:
done